framework/PageToolbar/PageToolbarFilter.tsx

```diff
@@ -295,6 +296,18 @@ function ToolbarFilterComponent(props: {
           queryErrorText={filter.queryErrorText}
           queryPlaceholder={filter.queryPlaceholder}
           isRequired={filter.isRequired}
+          footer={
+            <Button
+              variant="link"
+              onClick={() => {
+                filter?.openBrowse?.((selection) => {
+                  addFilter(selection);
+                }, filter.defaultSelection);
```

Comment:
It seems like the `openBrowse` should be taking in the current selection, which is loaded from the querystring values for the filter, instead of a filter.defaultSelection.

Reply:
Yes, filter default selection is now obsolete.